Code/GraphMol/ScaffoldNetwork/ScaffoldNetwork.h

```diff
@@ -173,5 +181,6 @@ RDKIT_SCAFFOLDNETWORK_EXPORT ScaffoldNetworkParams getBRICSNetworkParams();
 
 }  // namespace ScaffoldNetwork
 }  // namespace RDKit
+BOOST_CLASS_VERSION(RDKit::ScaffoldNetwork::ScaffoldNetwork, 1)
```

Comment:
swig doesn't like this (at least in Java)

Reply:
in the SubstructLibrary we moved these to a non swigged header file.

https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/SubstructLibrary/SubstructLibrarySerialization.h

Reply:
Yeah, but that works because the SubstructLibrary is using the non-intrusive serialization strategy. I wanted to avoid that here since the intrusive version is much simpler.
I will see if I can figure this out.